src/accelerate/accelerator.py

```diff
@@ -1717,6 +1717,9 @@ def load_state(self, input_dir: str):
         Args:
             input_dir (`str` or `os.PathLike`):
                 The name of the folder all relevant weights and states were saved in.
+            model_load_kwargs (`dict`):
+                Additional keyword arguments required for loading model, e.g., deepspeed's optional args for
+                `load_checkpoint`.
```

Comment:
```suggestion
                Additional keyword arguments required for loading model which are passed to the underlying load function, such as optional arguments for `DeepSpeed`'s `load_checkpoint` function.
```

Let's be a bit more explicit here